dashboard/app/views/layouts/application.html.haml

```diff
@@ -14,6 +14,10 @@
     = javascript_include_tag "js/#{js_locale}/common_locale"
     %script{src: minifiable_asset_path('js/essential.js')}
     %script{src: minifiable_asset_path('js/code-studio-common.js')}
+
+    //Included early to prevent JS Errors from being sent to New Relic
+    %script{src: minifiable_asset_path('js/layouts/application.js')}
+
```

Comment:
Here's an example of IE8 throwing when it tries to parse `application.js`:

![image](https://cloud.githubusercontent.com/assets/413693/25589398/59de064e-2e61-11e7-9637-c9a99d52d218.png)